### PR TITLE
[GH-756] Add delegates to ChannelCreateTx

### LINKED ISSUE #756: Add delegates in ChannelCreateTx
Blocked by: #755

### PR DESCRIPTION
**Pull request by cytadela8**
This made is on top of #789.

- added delegates to ChannelCreateTx
- added delegates to all channel opening functions
- modified tests to test delegates adding

This closes #756

### CODE REVIEW

File: apps/aecore/lib/aecore/channel/tx/channel_create_tx.ex [hunk 1]
```diff
@@ -55,6 +56,7 @@ defmodule Aecore.Channel.Tx.ChannelCreateTx do
   defstruct [
     :initiator_amount,
     :responder_amount,
+    :delegates,
```
Comment (grzegorz225) [inline apps/aecore/lib/aecore/channel/tx/channel_create_tx.ex]: Update docs

File: apps/aecore/lib/aecore/channel/updates/channel_create_update.ex [hunk 2]
```diff
@@ -45,6 +45,7 @@ defmodule Aecore.Channel.Updates.ChannelCreateUpdate do
     :initiator_amount,
     :responder,
     :responder_amount,
+    :delegates,
```
Comment (grzegorz225) [inline apps/aecore/lib/aecore/channel/updates/channel_create_update.ex]: Add delegates to docs

**Review by gorbak25 — COMMENTED** [review]
Looks good

**Review by thepiwo — DISMISSED** [review]
seems good

**Review by DanielaIvanova — COMMENTED** [review]
Looks good.

**Review by d-velev — DISMISSED** [review]

**Review by thepiwo — APPROVED** [review]

**Review by gorbak25 — APPROVED** [review]

**Review by DanielaIvanova — APPROVED** [review]

**Review by d-velev — APPROVED** [review]

**Pull request merged**